lib/src/test/java/com/cloudbees/groovy/cps/sandbox/SandboxInvokerTest.java

```diff
@@ -671,4 +671,19 @@ public void sandboxInterceptsImplicitCastsArrayAssignment() throws Throwable {
                 "LinkedHashMap.asBoolean()");
     }
 
+    @Test public void dynamicMethodPointer() throws Throwable {
+        assertIntercept(
+                "def method3() {\n" +
+                "    true\n" +
+                "}\n" +
+                "def mp = this.&/method${1 + 2}/\n" +
```

Comment:
This already worked in regular Groovy, in `groovy-sandbox`, and in `@NonCPS` methods with `groovy-cps`, so the fix only matters for CPS-transformed method pointers.